recipes/natu/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "natu" %}
+{% set version = "0.1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: c9302d52d3c01a75c18ba74d80ab9dc2ede9bb159a5c014641da16c0d754f7cc
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+
+test:
+  imports:
+    - natu
+
+about:
+  home: http://kdavies4.github.io/natu/index.html
+  license_file: LICENSE.txt
```

Comment:
Please add license and license_family entries

Reply:
I didn't know what to include for those. The license text doesn't make reference to any particular named license or license family that I know of. https://github.com/kdavies4/natu/blob/master/LICENSE.txt

Reply:
This looks like a modified BSD license.  However, the last clause makes me think we can't redistribute this:

```
This software is controlled under the jurisdiction of the United States
      Department of Commerce and subject to Export Administration Regulations.
      By downloading or using the Software, you are agreeing to comply with
      U. S. export controls.  Diversion contrary to law is prohibited.  The
      software cannot be exported or reexported to sanctioned countries that
      are controlled for Anti-Terrorism (15 CFR Part 738 Supplement 1) or to
      denied parties
```

CC @conda-forge/core This is something that would be have been caught by https://github.com/conda-forge/conda-smithy/pull/1256

Reply:
I have added `BSD` for both `license` and `license_family`. Leaving aside for the moment the issue of whether this package is redistributable, are these the correct values? Is there some way to indicate in the `license` value that the license is a _modified_ BSD, and not just BSD?